Liu-Yu-hw5.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
```

Comment:
1.	close, your output file has the two desired columns but they are not quite sorted properly. Your file is sorted based on years experience but not gender. The file we're looking for is sorted by gender and then for each gender by years experience. Additionally your file is separated by commas and the the question is asking for a file separated by spaces. Here's some code that would produce the file we're looking for cut -d , -f 1,2 wages.csv | grep -v "gender" | sort -t , -k1,1 -k2,2n | uniq | tr ',' ' '
a.	0.8
2.	1
3.	Good code, it doesn’t grab the lowest wage for those with 16 years of school  because your sort command doesn’t specify that you’re doing a numeric sort so the current results of that sort are incorrect. The difference in minimum wage between the two groups should be about $4
a.	0.7

2.5/3